zookeeper-docs/src/main/resources/markdown/zookeeperAdmin.md

```diff
@@ -1776,6 +1776,16 @@ and [SASL authentication for ZooKeeper](https://cwiki.apache.org/confluence/disp
     Specifies whether Online Certificate Status Protocol is enabled in client and quorum TLS protocols.
     Default: false
 
+* *ssl.tcnative.ocsp* and *ssl.quorum.tcnative.ocsp* :
```

Comment:
In think we should rename this to `ssl.ocsp.stapling` and `ssl.quorum.ocsp.stapling`. As you mention in the description, the feature is only available when native OpenSSL library is in use, otherwise setting this is a no-op.

Reply:
I went for ssl.tcnative.ocsp.stapling for now, LMK if that's acceptable to you, @anmolnar .

Reply:
I still don't see why we should add more fetures to a bugfix ticket.
You said that you oppose cleanup tickets, yet want to cram only tangetially related fixes to this ticket.
I can remove the new property for this ticket if you prefer.

Reply:
The new server only properties would be the first of their kind, but we can add them.
We can indeed document that they work differently from all the other properties.

I personally think that setting those properties on the server JDK command line is a better option though.